src/Fable.React/Fable.Helpers.ReactServer.fs

```diff
@@ -14,39 +14,36 @@ let private unitlessCssProps = System.Collections.Generic.HashSet<_>([ "animatio
 let private voidTags = System.Collections.Generic.HashSet<_>(["area"; "base"; "br"; "col"; "embed"; "hr"; "img"; "input"; "keygen"; "link"; "menuitem"; "meta"; "param"; "source"; "track"; "wbr"])
 
 // Adapted from https://github.com/facebook/react/blob/37e4329bc81def4695211d6e3795a654ef4d84f5/packages/react-dom/src/server/escapeTextForBrowser.js#L49
-let escapeHtml (sb:StringBuilder) (str: string) =
+let escapeHtml (sb:TextWriter) (str: string) =
   if isNull str then () else
   for c in str.ToCharArray() do
     match c with
-    | '"' -> sb.Append("&quot")
-    | '&' -> sb.Append("&amp;")
-    | ''' -> sb.Append("&#x27;") // modified from escape-html; used to be '&#39'
-    | '<' -> sb.Append("&lt;")
-    | '>' -> sb.Append("&gt;")
-    | c   -> sb.Append(c)
-    |> ignore
+    | '"' -> sb.Write("&quot")
+    | '&' -> sb.Write("&amp;")
+    | ''' -> sb.Write("&#x27;") // modified from escape-html; used to be '&#39'
+    | '<' -> sb.Write("&lt;")
+    | '>' -> sb.Write("&gt;")
+    | c   -> sb.Write(c)
 
-let inline private addUnit (html:StringBuilder) (key: string) (value: string) =
-  html.Append value |> ignore
+let inline private addUnit (html:TextWriter) (key: string) (value: string) =
+  html.Write value
   if not (unitlessCssProps.Contains key) then
-    html.Append "px" |> ignore
+    html.Write "px"
 
-let private cssProp (html:StringBuilder) (key: string) (value: obj) =
-  html.Append key |> ignore
-  html.Append ':' |> ignore
+let private cssProp (html:TextWriter) (key: string) (value: obj) =
+  html.Write key
+  html.Write ':'
 
   match value with
   | :? int as v -> addUnit html key (string v)
   | :? float as v -> addUnit html key (string v)
-  | _ -> escapeHtml html (value.ToString()) |> ignore
-
-  html.Append ';' |> ignore
```

Comment:
The cssProp function is not responsible for this anymore. It has been moved to renderHtmlAttr